lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -933,4 +933,32 @@ public static function illegalOverrideOfInheritedProperty($className, $propertyN
             )
         );
     }
+
+    /**
+     * @return self
+     */
```

Comment:
Let's use a native return type declaration here?

Reply:
we can do that in one big step when we introduce typehints more consistently, this is how all other methods in `MappingException`, i prefer the consistency for now. We can rector everything at some point